tools/docker/run.sh

```diff
@@ -89,10 +89,10 @@ main() {
 
     [ -z "$IPADDR" -a -n "$NETWORK" ] && {
         dbg "Generate random IP for container $NAME for network $NETWORK"
-        IPADDR=$(generate_container_random_ip $NETWORK)
+        IPADDR=$(generate_container_random_ip ${NETWORK})
```

Comment:
Same for the IP addresses: it would be better to rely on the docker-generated IP addresses instead of trying to make our own.